docs/conf.py

```diff
@@ -154,7 +157,7 @@ def setup(sphinx):
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-html_css_files = ["css/toggle.css"]
+html_css_files = ["css/toggle.css", "css/light.css"]
```

Comment:
Won't this apply `light.css` in both modes rather than only in the light mode as the name would suggest?

If your additions there were meant for both modes then I think that `custom.css` would be a better place.